src/main/java/com/exasol/projectkeeper/PomFileTemplateRunner.java

```diff
@@ -85,7 +88,7 @@ private boolean run(final Log log, final List<String> enabledModules, final PomT
         for (final PomTemplate template : TEMPLATES) {
             if (enabledModules.contains(template.getModule())) {
                 try {
-                    template.run(this.pom, runMode);
+                    template.run(this.pom, runMode, enabledModules);
```

Comment:
`run` doesn't say what the method actually does. It doesn't include any information at all

Reply:
Maybe Apply? Render? Validate? 
I also think that validation and applying should be divided into two independent parts